crates/biome_js_analyze/src/semantic_analyzers/complexity/no_useless_fragments.rs

```diff
@@ -257,7 +257,7 @@ impl Rule for NoUselessFragments {
                         jsx_tag_expression(AnyJsxTag::JsxSelfClosingElement(node)).into_syntax(),
                     ),
                     AnyJsxChild::JsxText(text) => {
-                        let new_value = format!("\"{}\"", text.value_token().ok()?);
+                        let new_value = format!("\"{}\"", text.value_token().ok()?.text().trim());
```

Comment:
```suggestion
                        let new_value = format!("\"{}\"", text.value_token().ok()?.text_trimmed());
```

Reply:
I tried it, but `_trimmed` method didn't trim '\n' in this case.
```
e.g.
text.value_token().ok()?.text_trimmed()
text.value_token().ok()?.token_text_trimmed()
```

Reply:
Oh I see. The newline is part of the string.